doc/makefile

```diff
@@ -16,7 +16,7 @@ DOXYGEN_CONF = CodeDocumentation.conf
 # doxygen uses: graphviz, latex
 html: $(DOXYGEN_CONF)
 	@# Generate the html documentation
-	@doxygen $(DOXYGEN_CONF)
+	@( cat $(DOXYGEN_CONF) ; echo "$(MFEM_DOXY_FLAGS)" ) | doxygen -
```

Comment:
The use case here is something like
```
make MFEM_DOXY_FLAGS="HAVE_DOT = NO"
```
(dot really slows down the documentation generation).

Reply:
I see this slowdown on Mac with recent versions of `graphviz/dot` as well. It used to be much faster. Also, in github actions the build is relatively fast (1m 1s), so `Ubuntu 18.04.6 LTS` does not seem affected by this slowdown yet.